contrib/aws/deis.template.json

```diff
@@ -399,6 +495,14 @@
     "DNSName" : {
       "Description" : "DNS Name of the ELB",
       "Value" :  { "Fn::GetAtt" : ["DeisWebELB", "DNSName"] }
-    }
+    },
+    "PublicDNSName": { "Fn::If": [
```

Comment:
You don't need the `Fn::If` here, you can just add a `Condition` field to this output.